### PR TITLE
Updates to package and release pipelines

### PR DESCRIPTION
**Pull request by adityapatwardhan**
# PR Summary

Create new release pipeline in OneBranch and update packaging and build pipeline to fix issues.

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.3/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

File: .pipelines/PowerShell-Release-Official.yml [hunk 1]
```diff
@@ -0,0 +1,419 @@
+#################################################################################
+#                         OneBranch Pipelines - Official                        #
+# This pipeline was created by EasyStart from a sample located at:              #
+#   https://aka.ms/obpipelines/easystart/samples                                #
+# Documentation:  https://aka.ms/obpipelines                                    #
```
Comment (TravisEz13) [inline .pipelines/PowerShell-Release-Official.yml:5]: offline comment

File: .pipelines/PowerShell-Release-Official.yml [hunk 2]
```diff
@@ -0,0 +1,419 @@
...
+
+      - template: /.pipelines/templates/release-validate-fxdpackages.yml@self
+        parameters:
+          jobName: 'linuxfxd'
+          displayName: 'Validate Linux Fxd Packages'
+          jobtype: 'linux'
+          artifactName: 'drop_linux_package_fxdependent'
+          packageNamePattern: '**/*linux-x64-fxdependent.tar.gz'
+
+      # defer task to run in Arm64
```
Comment (TravisEz13) [inline .pipelines/PowerShell-Release-Official.yml:178]: Either add more comments about why this is still comment or remove.

File: .pipelines/templates/release-create-msix.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        & $makeappx bundle /d $sourceDir /p "$outputDir\$pkgName"
+
+        Get-ChildItem -Path $sourceDir -Recurse
+        $vstsCommandString = "vso[task.setvariable variable=BundleDir]$outputDir"
+        Write-Host "sending " + $vstsCommandString
+        Write-Host "##$vstsCommandString"
+      displayName: Create MsixBundle
+      retryCountOnTaskFailure: 1
+
+    # Upload using
```
Comment (TravisEz13) [inline .pipelines/templates/release-create-msix.yml:79]: remove or comment why

File: .pipelines/templates/release-validate-packagenames.yml [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+          {
+              $messageInstance = "$($_.Name) is not a valid package name"
+              $message += $messageInstance
+              Write-Warning $messageInstance
+          }
+      }
+      if($message.count -gt 0){throw ($message | out-string)}
+    displayName: Validate Deb Package Names
+
+# Disable this til we fix the SBOM
```
Comment (TravisEz13) [inline .pipelines/templates/release-validate-packagenames.yml:131]: offline comment

File: .pipelines/templates/release-create-msix.yml (hunk 3 above)
Comment (adityapatwardhan) [reply .pipelines/templates/release-create-msix.yml:79]: Fixed. was a TODO and forgot about it

File: .pipelines/templates/release-validate-packagenames.yml (hunk 4 above)
Comment (adityapatwardhan) [reply .pipelines/templates/release-validate-packagenames.yml:131]: updated

File: .pipelines/PowerShell-Release-Official.yml (hunk 1 above)
Comment (adityapatwardhan) [reply .pipelines/PowerShell-Release-Official.yml:5]: removed

File: .pipelines/templates/release-create-msix.yml (hunk 3 above)
Comment (adityapatwardhan) [reply .pipelines/templates/release-create-msix.yml]: fixed

File: .pipelines/PowerShell-Release-Official.yml (hunk 2 above)
Comment (adityapatwardhan) [reply .pipelines/PowerShell-Release-Official.yml]: added a validation on arm64

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**